src/frontend/komponenter/Felleskomponenter/TilgangModal/TilgangModal.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import 'react-pdf/dist/Page/AnnotationLayer.css';
+import UIModalWrapper from '../Modal/UIModalWrapper';
+import { Knapp } from 'nav-frontend-knapper';
+import { Normaltekst } from 'nav-frontend-typografi';
+import IkkeOppfylt from '../../../ikoner/IkkeOppfylt';
+import { adressebeskyttelsestyper } from '../../../typer/oppgave';
+
+interface ITilgangModalProps {
+    åpen: boolean;
+    onRequestClose: () => void;
+    adressebeskyttelsegradering: string;
+}
+
+const TilgangModal: React.FC<ITilgangModalProps> = ({
+    åpen,
+    onRequestClose,
+    adressebeskyttelsegradering,
+}) => {
+    return (
+        <UIModalWrapper
+            modal={{
+                tittel: 'Diskresjonskode',
+                lukkKnapp: false,
+                visModal: åpen,
+                actions: [
+                    <Knapp
+                        key={'Avbryt'}
+                        mini={true}
+                        onClick={onRequestClose}
+                        children={'Avbryt'}
```

Comment:
Hva skal man avbryte? Det er vel heller en Lukk knapp? 🤷‍♂️ 

Og hvorfor skal den ikke ha lukknapp=true?

Reply:
Det ble vel sånn fordi skissene viste avbryt. Men når vi fjernet/utsatte muligheten for å tildele annen SB så ble det nok kanskje litt ulogisk å la avbryt-knappen være igjen alene. Bør nok derfor endres til bare lukk ja.